### ISSUE TITLE
EBSCO RIS data with multiple slashes

### ISSUE THREAD

**Issue by owcz** [zotero/translators#982]
On some EBSCO citations, such as [this](http://search.ebscohost.com/login.aspx?direct=true&AuthType=ip,uid&db=aph&AN=15089118&site=ehost-live&scope=site), the RIS spits back this for the date: 
```
Y1  - 2004/11//11/1/2004
```
on which Zotero's export chokes and just gives the year, `2004`, instead of November 2004. So this is again a publisher problem but it happens on a number of imports. Would it be smart to equip the EBSCO translator to handle these? Some other examples from a recent import:
 - `2005/07//July/August 2005` from [Choice](http://search.ebscohost.com/login.aspx?direct=true&AuthType=ip,uid&db=brd&AN=518481586&site=ehost-live&scope=site)
 - `2005/07//Jul/Aug2005` from [Choice](http://search.ebscohost.com/login.aspx?direct=true&AuthType=ip,uid&db=brd&AN=518481586&site=ehost-live&scope=site)
 - `2005/02//2/1/2005` from [Magill Book Reviews](http://search.ebscohost.com/login.aspx?direct=true&AuthType=ip,uid&db=aph&AN=16278846&site=ehost-live&scope=site)
 - `2005/01//Jan/Feb2005` from [Ruminator](http://search.ebscohost.com/login.aspx?direct=true&AuthType=ip,uid&db=hlh&AN=16254944&site=ehost-live&scope=site)

**Comment by adam3smith** [comment]
Yeah, I think this looks like a very good candidate for clean-up, just going with the date before the double slash. I'd think something like running
```
if text.match(/(Y1  - \d{4}.+?)\/\//){
  text = text.match(/(Y1  - \d{4}.+?)\/\//)[1]
}
```
(untested). I don't care much about the details as long as they a) make sure we have some type of date before the double slash and b) don't break things for dates without a double slash. 

I don't have EBSCO access at this time, so would be very happy to take a pull request from someone (you?) who does and cant tests this out a little.
